release-calendar/src/client/stylesheets/app.scss

```diff
@@ -0,0 +1,44 @@
+.main-container {
+    padding: 12px;
+    padding-top: 24px;
+}
+
+@mixin container-shadow {
+    box-shadow:
+        rgba(0, 0, 0, 0.2) 0px 2px 4px -1px,
+        rgba(0, 0, 0, 0.14) 0px 4px 5px 0px,
+        rgba(0, 0, 0, 0.12) 0px 1px 10px 0px;
+    transition: box-shadow 280ms cubic-bezier(0.4, 0, 0.2, 1);
+    display: block;
+    position: relative;
+    padding: 16px;
+    border-radius: 4px;
+}
+
+.col-side-pannel {
```

Comment:
typo: panel not pannel

Reply:
oop

Reply:
I've totally read that like a million times without noticing. yeet